src/ast/ast_validations.c

```diff
@@ -529,24 +529,24 @@ static AST_Validation _Validate_MATCH_Clause_Filters(const cypher_astnode_t *cla
 }
 
 static AST_Validation _Validate_CALL_Clauses(const AST *ast) {
-	/* Make sure procedure calls are valid:
+	/* make sure procedure calls are valid:
```

Comment:
Please update comment to include new validation,
Add a "sub-title" comment before each individual validation section:
```c
//------------------------------------------------------------------------------
// validate procedure exists
//------------------------------------------------------------------------------
...
//------------------------------------------------------------------------------
// validate argument count
//------------------------------------------------------------------------------
...
```